node/src/components/consensus/era_supervisor.rs

```diff
@@ -244,7 +248,7 @@ where
         );
 
         let slashed = era_id
-            .iter_other(self.bonded_eras)
+            .iter_other(self.initial_era.0, self.bonded_eras)
```

Comment:
So we will always use the `initial_era`? What if we are in era 200 and initial era is still 0? Are we going to keep iterating from `Era(0)` forever?

Reply:
Not really - it will always iterate through at most `self.bonded_eras` of eras. The initial era number is just so that the lowest era number returned by the iterator doesn't go below it.

Reply:
Ah, you always add an offset to the initial era.